include/boost/json/detail/impl/number.ipp

```diff
@@ -215,7 +223,7 @@ loop:
             BOOST_ASSERT(dig_ == 0);
             ++p;
             pos_ = 0;
-            st_ = state::zeroes;
+            st_ = state::mant_frac_1_digit;
```

Comment:
Suggested: `state::mantf`

Reply:
may I at least comment the purpose of the state?